doc/user-guide/user_guide.md

```diff
@@ -1,71 +1,41 @@
-# Supported Dialects
+#User Guide
```

Comment:
Link to this user guide from README. You can remove the link later when the documentation is updated